.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] I noticed that Eslint is not checked green because it is missing in your PR  😢  .  For this project it required to set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository. Please make sure to fix it in order to make the app works as expected 👍